src/pages/hardware/developingAnIOX/messagingProtocol.tsx

```diff
@@ -0,0 +1,1361 @@
+                <tr>
+                    <td>0</td>
+                    <td>Number of Beeps</td>
+                </tr>
+                <tr>
+                    <td>1</td>
+                    <td>Duration (Multiple of 56ms)</td>
+                </tr>
+                <tr>
+                    <td>2</td>
+                    <td>Delay Between Beeps (Multiple of 56ms)</td>
+                </tr>
+            </tbody>
+        </table>
+        <h3>IOX Request/Status (0x25)</h3>
+        <p>Sent from the IOX to the GO device to inform the GO device of events or status changes.</p>
+        <h4>Payload</h4>
+        <table>
+            <thead>
+                <tr>
+                    <th>Byte #</th>
+                    <th>Byte Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0-1</td>
+                    <td>Information Type</td>
+                </tr>
+                <tr>
+                    <td>2-7</td>
+                    <td>Optional bytes that depend on the type</td>
+                </tr>
+            </tbody>
+        </table>
+        <h3>Information Type 0 - Busy</h3>
+        <p>This message indicates to the GO device that the issuing IOX is busy with a critical task and that the GO should not enter the sleep state. The IOX should send this message again to release the GO device once it has completed its critical tasks.</p>
+        <table>
+            <thead>
+                <tr>
+                    <th>Parameter Type</th>
+                    <th>Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0-1</td>
+                    <td>0x0000</td>
+                </tr>
+                <tr>
+                    <td>2</td>
+                    <td>
+                        0 = Not busy
+                        <br></br>
+                        1 = Busy
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+        <h3>Information Type 1 - Packet Wrapper</h3>
+        <p>This is used to send a packet of up to 1023 bytes of binary data through the GO device to MyGeotab.</p>
+        <p>Use cases:</p>
+        <ol>
+            <li>Send Packet Wrapper - Beginning of data packet (0).</li>
+            <li>Multiple RX Data (0x0C) messages until the entire bypass binary data is sent. Pay attention to the Acknowledge (0x14) message responded from GO if the GO Receive-Buffer is ready to accept the next RX Data (0x0C) message. The streaming flow control bit in the ACK is not relevant for this type of exchange and can be ignored.</li>
+            <li>Send Packet Wrapper - End of data packet (1).</li>
+            <li>At the end, GO sends confirmation with a Application Specific Data (0x1C) Type 0 (Modem transmission result) message to indicate if the packet has been accepted within 6 seconds.</li>
+        </ol>
+        <table>
```

Comment:
we should have tables embedded in a div with the .table-container class until we come up with the final styling so it can be applied later on